cmd/registry/cmd/apply/testdata/sample/artifacts/manifest.yaml

```diff
@@ -13,6 +13,7 @@ data:
         - pattern: $resource.spec
           filter: mime_type.contains('openapi')
       action: registry compute lint $resource.spec --linter spectral
+      refresh: null
```

Comment:
This is needed because [apply_test.go](https://github.com/apigee/registry/blob/main/cmd/registry/cmd/apply/apply_test.go#L104) uploads an artifact from this yaml and then exports it to check if the exported artifact is the same. 

While exporting, the export artifact logic is using this [option](https://github.com/apigee/registry/blob/main/cmd/registry/patch/artifact.go#L123) which exports `refresh:null` fields in the yaml. If the exported yaml doesn't match the one defined in the file, the test fails. Hence, I added these to make sure `apply_test.go` is passing.

@timburks do you think there can be a better workaround to handle this test?

Reply:
Setting "EmitUnpopulated" addressed a problem where zero-valued integers (display order in lifecycle and taxonomies) weren't being written. I don't think we can get different behavior for different fields without adding per-resource custom code (which we've minimized). I agree that explicitly including this here is verbose and unnecessary for reimport, and it does raise the question "what is this?" but that seems fine (maybe beneficial even) in an example.

tl;dr I'm ok with keeping this default field in this sample manifest.

Reply:
Sounds like EmitUnpopulated is needed. The test verification might have room for improvement though. I'm not sure we need to be failing a test if the export YAML isn't identical to the input, as long as we make sure they're semantically equivalent.

I'm fine with this until (if ever) we change the test verification.